Documents/Automation/EncryptRootVolume/Documents/npark-encryptrootvolume.json

```diff
@@ -15,10 +15,24 @@
       "type": "String",
       "description": "(Optional) The ARN of the role that allows Automation to perform the actions on your behalf.",
       "default": ""
+    },
+    "maxWaitForCreateSnapshot": {
+      "type": "Integer",
+      "description": "(Optional) maximum wait time for creating snapshot of the root volume (prior to encryption). Default maximum wait time is 30 minutes (1800 seconds)",
+      "default": 1800
+    },
+    "maxWaitForSnapshotWithEncryption": {
+      "type": "Integer",
+      "description": "(Optional) maximum wait time for copying snapshot (with encryption) of the root volume. Default maximum wait time is 30 minutes (1800 seconds)",
```

Comment:
Maximum (uppercase first letter), not maximum

Reply:
Corrected in the latest commit